layouts/partials/footer.html

```diff
@@ -1,6 +1,6 @@
 </main>
 <footer class="footer">
-  <span>&copy; {{ now.Year }} {{ .Site.Title }}</span>
+  <span>&copy; {{ now.Year }} <a href="{{ .Site.BaseURL }}">{{ .Site.Title }}</a></span>
```

Comment:
I suggest using `{{ "/" | absURL }}` instead of `{{ .Site.BaseURL }}` to maintain the consistency between URLs under different environments. :)

Reply:
Thank you for your suggestion ~